lib/manager/npm/post-update/npm.spec.ts

```diff
@@ -1,30 +1,26 @@
 import { exec as _exec } from 'child_process';
 import upath from 'upath';
-
 import { envMock, mockExecAll } from '../../../../test/exec-util';
-import { mocked } from '../../../../test/util';
-import * as _env from '../../../util/exec/env';
-import * as _fs from '../../../util/fs/proxies';
+import { env, fs } from '../../../../test/util';
 import * as npmHelper from './npm';
 
 jest.mock('child_process');
 jest.mock('../../../util/exec/env');
-jest.mock('../../../util/fs/proxies');
+jest.mock('../../../util/fs');
 jest.mock('./node-version');
 
 const exec: jest.Mock<typeof _exec> = _exec as any;
-const env = mocked(_env);
-const fs = mocked(_fs);
 
 describe('manager/npm/post-update/npm', () => {
   beforeEach(() => {
     jest.resetAllMocks();
     jest.resetModules();
     env.getChildProcessEnv.mockReturnValue(envMock.basic);
   });
+
   it('generates lock files', async () => {
     const execSnapshots = mockExecAll(exec);
-    fs.readFile = jest.fn(() => 'package-lock-contents') as never;
+    fs.readLocalFile = jest.fn(() => 'package-lock-contents') as never;
```

Comment:
```suggestion
    fs.readLocalFile.mockResolvedValueOnce(() => 'package-lock-contents');
```
Reuse mock functions